From b8e2a31938b8857e908c8ccfee89a06651389d6a Mon Sep 17 00:00:00 2001
From: rpm-build <rpm-build>
Date: Tue, 28 Jul 2015 11:35:20 +0200
Subject: [PATCH] Backport support in to the netns for non intel archs

Based on upstream netns commits:

https://github.com/vishvananda/netns/commit/604eaf189ee867d8c147fafc28def2394e878d25
https://github.com/vishvananda/netns/commit/cbbe6847e60878d8254b4d785cfbd656a8f56bda
https://github.com/vishvananda/netns/commit/b7a04d6db3dce92b370e6d5b6ab9e2361a3c53e1

Also adds support for ppc64.
---
 vendor/src/github.com/vishvananda/netns/netns_linux_arm.go   | 2 +-
 vendor/src/github.com/vishvananda/netns/netns_linux_arm64.go | 7 +++++++
 vendor/src/github.com/vishvananda/netns/netns_linux_ppc64.go | 7 +++++++
 vendor/src/github.com/vishvananda/netns/netns_linux_s390x.go | 7 +++++++
 4 files changed, 22 insertions(+), 1 deletion(-)
 create mode 100644 vendor/src/github.com/vishvananda/netns/netns_linux_arm64.go
 create mode 100644 vendor/src/github.com/vishvananda/netns/netns_linux_ppc64.go
 create mode 100644 vendor/src/github.com/vishvananda/netns/netns_linux_s390x.go

diff --git a/vendor/src/github.com/vishvananda/netns/netns_linux_arm.go b/vendor/src/github.com/vishvananda/netns/netns_linux_arm.go
index fbf165e..9c74eb5 100644
--- a/vendor/src/github.com/vishvananda/netns/netns_linux_arm.go
+++ b/vendor/src/github.com/vishvananda/netns/netns_linux_arm.go
@@ -3,5 +3,5 @@
 package netns
 
 const (
-  SYS_SETNS = 374
+  SYS_SETNS = 375
 )
diff --git a/vendor/src/github.com/vishvananda/netns/netns_linux_arm64.go b/vendor/src/github.com/vishvananda/netns/netns_linux_arm64.go
new file mode 100644
index 0000000..741a302
--- /dev/null
+++ b/vendor/src/github.com/vishvananda/netns/netns_linux_arm64.go
@@ -0,0 +1,7 @@
+// +build linux,arm64
+
+package netns
+
+const (
+  SYS_SETNS = 268
+)
diff --git a/vendor/src/github.com/vishvananda/netns/netns_linux_ppc64.go b/vendor/src/github.com/vishvananda/netns/netns_linux_ppc64.go
new file mode 100644
index 0000000..e97562f
--- /dev/null
+++ b/vendor/src/github.com/vishvananda/netns/netns_linux_ppc64.go
@@ -0,0 +1,7 @@
+// +build linux,ppc64
+
+package netns
+
+const (
+  SYS_SETNS = 350
+)
diff --git a/vendor/src/github.com/vishvananda/netns/netns_linux_s390x.go b/vendor/src/github.com/vishvananda/netns/netns_linux_s390x.go
new file mode 100644
index 0000000..cc13e62
--- /dev/null
+++ b/vendor/src/github.com/vishvananda/netns/netns_linux_s390x.go
@@ -0,0 +1,7 @@
+// +build linux,s390x
+
+package netns
+
+const (
+  SYS_SETNS = 339
+)
-- 
2.1.0

